contributors/design-proposals/local-storage-overview.md

```diff
@@ -512,7 +523,7 @@ Note: Block access will be considered as a separate feature because it can work
         requests:
           storage: 80Gi
     ```
-4. It is also possible for a PVC that requests `volumeType: file` to also use a block-based PV.  In this situation, the block device would get formatted with the filesystem type specified in the PV spec.  And when the PV gets destroyed, then the filesystem also gets destroyed to return back to the original block state.
+4. The block device would get formatted with the filesystem type specified in the PV spec.  And when the PV gets destroyed, then the filesystem also gets destroyed to return back to the original block state.
```

Comment:
No, this document is forward looking and contains ideas beyond the initial alpha phase.

I have a separate user guide document with the current state of what's implemented.